### ISSUE TITLE
App crashes every time I try to start it

### ISSUE THREAD

**Issue by unforgettableid** [pnemonic78/HalachicTimes#123]
Hi!

* I recently used the Google Play Store to upgrade from Halachic Times version 13.20 to version 13.22.
* I also used the [Dark Mode](https://play.google.com/store/apps/details?id=systems.maju.darkmode) app to change my phone from "day mode" to "night mode".

Now, I cannot start Halachic Times itself, and I cannot place any of the app's widgets on my home screen.  Whenever I try, I get a message:  "Unfortunately, Halachic Prayer Times has stopped."

A stack trace follows, at the end of this message.

I am using an old LG G5 running Android 8.0.0 "Oreo".

Thank you for reading this!

# Stack trace

```
Fatal exception: main
Process: net.sf.times
java.lang.NoClassDefFoundError: Failed resolution of: Lcom/github/times/appwidget/ZmanimWallpaperHelper;
       at com.github.times.ZmanimApplication.<init>(SourceFile:4)
       at java.lang.Class.newInstance(Native Method)
       at android.app.Instrumentation.newApplication(Instrumentation.java:1102)
       at android.app.Instrumentation.newApplication(Instrumentation.java:1087)
       at android.app.LoadedApk.makeApplication(LoadedApk.java:1011)
       at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5905)
       at android.app.ActivityThread.-wrap1(Unknown Source:0)
       at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1680)
       at android.os.Handler.dispatchMessage(Handler.java:105)
       at android.os.Looper.loop(Looper.java:164)
       at android.app.ActivityThread.main(ActivityThread.java:6710)
       at java.lang.reflect.Method.invoke(Native Method)
       at com.android.internal.os.Zygote$MethodAndArgsCaller.run(Zygote.java:240)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:770)
Caused by: java.lang.ClassNotFoundException: com.github.times.appwidget.ZmanimWallpaperHelper
       at java.lang.VMClassLoader.findLoadedClass(Native Method)
       at java.lang.ClassLoader.findLoadedClass(ClassLoader.java:738)
       at java.lang.ClassLoader.loadClass(ClassLoader.java:363)
       at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
       ... 14 more
Caused by: java.lang.NoClassDefFoundError: Failed resolution of: Landroid/app/WallpaperManager$OnColorsChangedListener;
       ... 14 more
Caused by: java.lang.ClassNotFoundException: Didn't find class "android.app.WallpaperManager$OnColorsChangedListener" on path: DexPathList[[zip file "/data/app/net.sf.times-bTORDDrREtrDejfsvcB9OA==/base.apk"],nativeLibraryDirectories=[/data/app/net.sf.times-bTORDDrREtrDejfsvcB9OA==/lib/arm64, /system/lib64, /system/vendor/lib64]]
       at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:93)
       at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
       at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
       ... 14 more

**Comment by pnemonic78** [comment]
fixed in commit 5d5adbfcaee7cd5f35b0f1b147a392355af93df5

**Comment by unforgettableid** [comment]
Verified fixed in version 13.24.

I appreciate your support for users of old Android versions!
